cmd/getConfig.go

```diff
@@ -158,14 +158,24 @@ func generateConfig(utils getConfigUtils) error {
 
 		stepConfig, err = myConfig.GetStepConfig(flags, GeneralConfig.ParametersJSON, customConfig, defaultConfig, GeneralConfig.IgnoreCustomDefaults, paramFilter, params, metadata.Spec.Inputs.Secrets, resourceParams, GeneralConfig.StageName, metadata.Metadata.Name, metadata.Metadata.Aliases)
 		if err != nil {
-			return errors.Wrap(err, "getting step config failed")
+			return stepConfig, errors.Wrap(err, "getting step config failed")
 		}
+		log.Entry().Infof("getConfig printing stepConfig: %v", stepConfig)
```

Comment:
maybe make this a debug message instead ...

Reply:
I've removed it